Version 0.2.0 causes Starlette to raise low-level exception about bytes instead of str

Trying to pull in version 0.2.0 causes several of our tests to fail with this low-level exception raised within Starlette when processing headers:

```
../.cache/pypoetry/virtualenvs/excel-processor-Io29J3C4-py3.10/lib/pypy3.10/site-packages/starlette/testclient.py:317: in send
    raw_kwargs["headers"] = [(key.decode(), value.decode()) for key, value in message.get("headers", [])]
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

.0 = <sequenceiterator object at 0x00007ff3eaeebe08>

>   raw_kwargs["headers"] = [(key.decode(), value.decode()) for key, value in message.get("headers", [])]
E   AttributeError: 'str' object has no attribute 'decode'. Did you mean: 'encode'?
```

Looks like Starlette assumes (rightly or wrongly, I haven't looked closer) that keys and values in the message headers will always be `bytes` and not `str`, and that ceased to be true with https://github.com/arni-inaba/asgi-request-id/pull/4/files#diff-ac0549573b02fd048abe703bb7d7b90d7810308f6ddeddeba0c19936545ce362L33-R37

--- a/asgi_request_id/middleware.py
+++ b/asgi_request_id/middleware.py
@@ -34,7 +34,7 @@ class RequestIDMiddleware:
         def send_wrapper(response):
             response_headers = response.get("headers")
             if response_headers:
-                response["headers"].append((self.outgoing_id_header, request_id.encode()))
+                response["headers"].append((self.outgoing_id_header.encode(), request_id.encode()))
             return send(response)
 
         if scope["type"] == "http":

